x-pack/plugins/apm/public/components/app/service_inventory/ServiceList/index.tsx

```diff
@@ -185,6 +71,150 @@ const SERVICE_HEALTH_STATUS_ORDER = [
 export function ServiceList({ items, noItemsMessage }: Props) {
   const displayHealthStatus = items.some((item) => 'healthStatus' in item);
 
+  const showTransactionTypeColumn = items.some(
+    ({ transactionType }) =>
+      transactionType !== TRANSACTION_REQUEST &&
+      transactionType !== TRANSACTION_PAGE_LOAD
+  );
+
+  const SERVICE_COLUMNS: Array<ITableColumn<ServiceListItem>> = [
```

Comment:
Is there a good reason to move it inline? If it needs `showTransactionTypeColumn`, maybe a function is better here (as this is already a long render function)?